pkg/trace/api/api.go

```diff
@@ -188,29 +205,40 @@ func (r *HTTPReceiver) attachDebugHandlers(mux *http.ServeMux) {
 	mux.Handle("/debug/vars", expvar.Handler())
 }
 
-// Listen creates a new HTTP server listening on the provided address.
-func (r *HTTPReceiver) Listen(addr, logExtra string) error {
-	listener, err := net.Listen("tcp", addr)
-	if err != nil {
-		return fmt.Errorf("cannot listen on %s: %v", addr, err)
+// listenUnix returns a *net.Listener listening on the given "unix" socket path.
```

Comment:
`*net.Listener` should be `net.Listener`?

Reply:
Indeed.